### DISCUSSION TITLE
How to download a file without header?

### DISCUSSION THREAD

**Discussion by GitHub30** [deta/cloud-docs#434] [Help] [answered]
How to download a file without header?

```bash
wget --header="X-Api-Key: xxxxxxxxxx" https://drive.deta.sh/v1/c0m8f10j/Drive/files/download?name=eevee.jpg
```

```html
<img src="https://drive.deta.sh/v1/c0m8f10j/Drive/files/download?name=eevee.jpg">
```

workaround?
https://github.com/GitHub30/deta-drive-proxy
https://public.deta.dev/c0m8f10j/Drive/eevee.jpg

**Comment by abdelhai** [comment, marked as answer]
you mean how to download a file from drive without an api key? this is not possible. a file server like the one you provided is the solution.
deta drive itself is a storage service and doesn't have a server.

also please remove the project key you provided. anyone can now steal and modify your project's data. cc @aavshr
